### ISSUE TITLE
Accessibility bug: Cannot reach the menu on the left or right unless I remove all feeds

### ISSUE THREAD

**Issue by SleeplessByte** [bluesky-social/social-app#6370]
### Steps to Reproduce

1. Go to https://bsky.app/
2. Try to tab your way into the menu(s)
3. Good luck getting to the end of the feed

There are several issues here: 

- The focus order does not match the visual order.
- There is no way to quickly go past a post (`jk` navigation for example).
- There is no way to skip sections of the content (no skip links).

Solving the focus order solves part of it as the right sidebar on bluesky, just like on Twitter and LinkedIn is mostly useless.

- [ ] Ensure the [focus order matches visual order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order)
- [ ] Ensure the user can _skip_ the navigation
- [ ] Ensure the user can _skip_ the main content (to get to the sidebar on the right)

This is related to #6369 and the technique for skipping is the same.

You probably want several skip links, [understanding bypass blocks](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html). Usually these links are [added to skip the navigation to main content](https://webaim.org/techniques/skipnav/), which is probably what you want here as well, and you can add a second one to skip the feed to get to the right sidebar.

> [!CAUTION]
> There is no consensus on how to deal with a bottom navigation bar, but the people I test my software with are all okay with that taking focus precedence over the rest of the page, even though visually it comes "last". They say things like "yes, but a lot of apps have such a navigation so it's okay". My advice here is not leading and there is no WCAGuideline I can link to help here. You are just going to have to make a choice and defend it or be willing to adapt based on real-user-feedback.

The bottom navigation bar currently sits at the end of the focus order. I think this should move to the front, and in fact, that's how I always implement it on the products I work on. You don't want to have to scroll through the feed to get to the navigation.

### Attachments

| Breakpoint | Focus Order |
|------------|-------------|
| Desktop | ![Focus order on a logged-in home page showing to reach the navigation you need to tab over 600 times, desktop viewport](https://github.com/user-attachments/assets/16b62312-eab5-42e3-aa60-e2038386c016)
| Small laptop / Tablet | ![Focus order on a logged-in home page showing to reach the navigation you need to tab over 600 times, tablet viewport](https://github.com/user-attachments/assets/85c6ebbb-5cbf-4fe9-a8e9-b2aea1dd9889)
| Mobile | ![Focus order on a logged-in home page showing to reach the navigation you need to tab over 600 times, mobile viewport](https://github.com/user-attachments/assets/152ad6c2-da1f-40a5-9acd-ea08a88cf05b)

### What platform(s) does this occur on?

Web (Desktop), Web (Mobile)

### Device Info

Not Relevant

### What version of the app are you using?

Web

### Additional Information

Okay so on Twitter this is *partially* mitigated because the Focus order isn't broken and goes from left (menu) to feed to right sidebar. However, I need to hold `j` to get to the bottom of the loaded posts followed by `tab` to get to the sidebar before the next batch loads. So it's not optimal. When browsing with a mobile user agent, there is a bottom navigation bar that comes first in the focus order, which fixes it.

LinkedIn has the same challenge as Twitter, but again the menu is in focus order so it's not that much of a problem as the right sidebar is mostly useless there. They also have skip links at the top, which is how this should work.

![LinkedIn has skip links when navigating by keyboard that allow you to skip to search or to the main content (which is the feed and skips all the navigation)](https://github.com/user-attachments/assets/c88ef585-4958-4428-81e4-2bfa4860d606)
